onnxruntime/core/providers/plaidml/plaidml_ops.h

```diff
@@ -12,8 +12,8 @@
 #include "core/graph/onnx_protobuf.h"
 
```

Comment:
Why are we forward declaring these? I see a commented line above that includes `edsl.h`. Why not include and then drop the forwards?

Reply:
Agreed, and if all that's needed is to uncomment `edsl.h` and remove forward decs then let's do that. That said, if there are reasons this change is non-trivial, my opinion is that it's better to get this landed promptly than to get this cleaned.